libindy/src/api/mod.rs

```diff
@@ -257,5 +258,31 @@ pub extern fn indy_set_runtime_config(config: *const c_char) -> ErrorCode {
 
     trace!("indy_set_runtime_config: <<< res: {:?}", res);
 
+    res
+}
+
+/// Get details for last occurred error
```

Comment:
1. I believe we need to provide more information on when this method should be called. 
2. Today we voted to avoid error_hanlde
3. I don't like the idea of returning error code here. It can cause recursion calls to resolve errors
4. We need to add information about returned pointer lifetime to description.
5. We need to add information on when we resolve back-trace

Reply:
6. As i remembered we decided to rename it to get_current_error.